types/to-px/index.d.ts

```diff
@@ -0,0 +1,12 @@
+// Type definitions for to-px 1.1
+// Project: https://github.com/mikolalysenko/to-px
+// Definitions by: Vindhya Raviraj <https://github.com/vindhya>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare function toPX(str: string): number;
+
+declare namespace toPX {
```

Comment:
What's the namespace for?  It doesn't seem to be used in the tests or the sample usage in the readme.

Reply:
Apologies, it's my first time contributing and I thought it was necessary. I've removed the namespace declaration now.

Reply:
No worries.  Thanks for the update!